test/demo-logger-using-pvc/logger.py

```diff
@@ -0,0 +1,44 @@
+#! /usr/bin/python3
+
+import logging
+import socket
+import time
+import sys
+
+LOG_DIR = "/var/log/demoLogger/"
+HOSTNAME = socket.gethostname()
+LOG_FILE = LOG_DIR + HOSTNAME + ".log"
+
+def initialize_logger():
+    log = logging.getLogger("demo-logger")
+    log.setLevel(level=logging.DEBUG)
+    formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+    fh = logging.FileHandler(LOG_FILE)
+    fh.setLevel(level=logging.DEBUG)
+    fh.setFormatter(formatter)
+    log.addHandler(fh)
+    return log
+    
+def infi_logger(logger=None):
+    i = 0
+    while True:
+        logline = f"Logging count {i}"
+        if logger:
+            logger.info(logline)
+        else:
+            print(logline)
+        time.sleep(20)
```

Comment:
for demo, sleeping 20 seconds would be too much. Log every 5 seconds atleast

Reply:
addressed.